dbSearcher/configs/config.php

```diff
@@ -3,9 +3,9 @@
 /* Database Connection Credentials */
 /***********************************/
 
- $hostname  = 'localhost';    // The IP or DNS where the database server is located
- $username  = 'root';         // Username for login to the database server
- $password  = 'root';         // Password for login to the database server
+ $hostname  = 'hostname';    // The IP or DNS where the database server is located
+ $username  = 'username';         // Username for login to the database server
+ $password  = 'password';         // Password for login to the database server
```

Comment:
This is cool! However can be put in its own PR, be sure to have the same spacings inbetween the comments as well!